js/prototype/prototype.js

```diff
@@ -1111,7 +1111,7 @@ function $w(string) {
   return string ? string.split(/\s+/) : [];
 }
 
-Array.from = $A;
+Array.from = Array.from || $A;
```

Comment:
genuine question because it's hard to test with a browser that doesn't have Array.from nowadays. if Array.from is not defined, won't it cause an error instead of just evaluating the second part of the "or" statement?

Reply:
answering myself, tested with

```
Array.from = undefined;
alert(Array.from);
Array.from = Array.from || $A;
```

and it works without generating errors

Reply:
Popular browsers since 2016 support `Array.from()`, so it's probably cleaner to just remove the polyfill in line 1114 `Array.from = $A`?

Reply:
to me it's fine either ways, I guess this one is the safest approach in order to modify the library the less possible